tests/queries/0_stateless/02174_cte_scalar_cache_mv.reference

```diff
@@ -39,7 +39,7 @@
 89	89	89	89	5
 94	94	94	94	5
 99	99	99	99	5
-02177_MV	0	0	22
+02177_MV	2	63	22
```

Comment:
Cool! Looks like this test is fixed.

Reply:
Actually, I don't understand why it was broken, but I understand what changed the behaviour.